app/controllers/FaciaToolController.scala

```diff
@@ -41,26 +41,41 @@ class FaciaToolController(
         Ok(Json.toJson(configJson)).as("application/json")}}}
 
   def getCollection(collectionId: String) = AccessAPIAuthAction.async { implicit request =>
-
     val collectionPriorities = configAgent.getFrontsPermissionsPriorityByCollectionId(collectionId)
 
     withModifyGroupPermissionForCollections(collectionPriorities, Set()) {
+      val collections = fetchCollections(List(collectionId))
 
-      FaciaToolMetrics.ApiUsageCount.increment()
-      val collection = frontsApi.amazonClient.collection(collectionId).flatMap { collectionJson =>
-        collectionJson.map(json => mediaServiceClient.addThumbnailsToCollection(json, collectionId)) match {
-          case Some(f) => f.map(Some(_))
-          case None => Future.successful(None)
-
-        }
-      }
-      collection.map(c => NoCache {
+      collections.head.map(c => NoCache {
         Ok(Json.toJson(c)).as("application/json")
       })
+    }
+  }
+
+  def getCollections(collectionIdsString: String) = AccessAPIAuthAction.async { implicit request =>
```

Comment:
Public functions should have return types ideally.

Reply:
👍 will do!